packages/core/src/views/account-center/Micro.svelte

```diff
@@ -0,0 +1,100 @@
+<script lang="ts">
```

Comment:
Is there a reason for renaming this file to `Micro.svelte`? It ruins the diff for what has changed on this file, making it harder to review.

Reply:
`Micro.svelte` is a copy of `Minimized.svelte` — the same file with a bit fewer style customizations. I left the initial `Minimized.svelte` because I wasn't sure if we would need it for future configurations.